Recreate original Alterac Valley.

ratkosrb opened the pull request:
## 🍰 Pullrequest
Using Korrak's Revenge event data to restore the original AV spawns that were removed in later vanilla patches.

### Todo / Checklist
- [X] Respawn Creatures
- [x] Respawn GameObjects
- [ ] Assign correct patch to everything
- [ ] Add missing scripts

ratkosrb committed:
Respawn all creatures.

ratkosrb committed:
Respawn all gameobjects.

Daribon committed:
Add 42 missing mining nodes.

Daribon commented:
Since there is just so much things to consider here , I'm gonna start writing down some notes (mainly for myself.)

From this PR:
![WoW_1dgEzoK7yU](https://github.com/user-attachments/assets/0cfa2f2e-2786-4e92-8d9b-a412c0bc4dfd)
The stable at SH bunker currently has stabled rams spawned. They should not be pre spawned.
I think the reason they are currently spawned is because original korrak sniffs, someone had turned in rams up in north which apparently does cause them to spawn even at SH bunker. The later korrak sniffs from SL which doesn't stable rams in dun baldar, does not have rams at SH bunker.

In regards to ram stable in Dun Baldar, seems like it's supposed to have ram spawn outside the building too:
![image](https://github.com/user-attachments/assets/db47392c-50e2-4628-b640-a72cf31cbb26)
Image from 7th February 2006.

---

Missing flags at Winterhax Hold when Snowfall Graveyard is captured:
![WoW_3uTpBPiybE](https://github.com/user-attachments/assets/04522e4d-2d94-4aad-86e8-26d1d210af77)
Found in korrak revenge SL sniffs, not found in the original korrak revenge sniffs.

---

Weird object's all around Drek'thar and Vanndar in sniffs:
![WoW_4LQqfB2DLN](https://github.com/user-attachments/assets/4c2f9689-901a-45c2-805a-82b3f694f981)
![WoW_YdMvUuPdrn](https://github.com/user-attachments/assets/06bace43-fad3-412b-be84-e02a87817f8c)
Image from classic sniffs. They seem to be unused objets. Are they pre spawned? 
I can't remember seeing it being spawned before or is something that just happens to spawn upon Drek or Vanndars death?

---
Blizz seems to have messed up graveyard objects in Korrak's Revenge. They do not match 1.5.0 screenshots.
Confirmed wrong thus far is IB GY, SP GY, FW GY, SH GY
From this PR and in korrak sniffs sniff:
![WoW_QQAtVmJ4Ho](https://github.com/user-attachments/assets/b11565d3-4c43-40d9-aaff-6140cf4d773e)

FW GY:
![alteractal35](https://github.com/user-attachments/assets/b458fa58-f56f-4661-a91f-76b7b63fae77)
![alteractal37](https://github.com/user-attachments/assets/d2665bad-2191-4135-8887-5b8d0ac8c099)
![565 - ZBAY9](https://github.com/user-attachments/assets/4f57073b-f10a-4429-a148-52901a5ae7b8)

SH GY:
![WoWScrnShot_072005_012810](https://github.com/user-attachments/assets/aa8b79c2-03ad-4db9-bb0a-9b03c7813cda)
Dated 20th of July 2005.
![WoWScrnShot_060905_172349](https://github.com/user-attachments/assets/d09937d7-0fce-47d1-9be3-38a8dae0cce0)
6 September 2005

https://www.youtube.com/watch?v=KLfK6v4ATw0
Video from 27 06 2005

IB GY:
![172 - eA42y](https://github.com/user-attachments/assets/64183ff0-453a-41ae-927a-f8d56ede62d8)
Unknown exact date, somewhere between 1.5.0 and 1.8.0. Graveyard flag not seen here it exists in korrange revenge sniffs or in 1.12.1.
![191 - GQOYr](https://github.com/user-attachments/assets/519d0f22-b28c-472b-b766-d933cc7776b5)
![212 - BhOIM](https://github.com/user-attachments/assets/796f8da2-2dbe-42d6-9ede-f746fb4936d6)
Unknown exact date, somewhere between 1.5.0 and 1.8.0. Graveyard flag not seen here it exists in korrange revenge sniffs or in 1.12.1.
![185 - FA3gZ](https://github.com/user-attachments/assets/b40265d0-62ec-4f84-b176-beae083e17c7)
![447 - T5sr8](https://github.com/user-attachments/assets/70a6f13e-84a1-434b-a2ee-a50d9bc89e39)
![192 - 7zl6u](https://github.com/user-attachments/assets/bcedadd0-eba2-4b6e-b19d-6778a6b237d1)
![0422222](https://github.com/user-attachments/assets/0977ba1e-a06e-42e9-b014-b5811e3bb0b2)
![04011111](https://github.com/user-attachments/assets/c768366b-a557-48fb-a111-2af1561ac520)
![alteractal10](https://github.com/user-attachments/assets/7a5bf21d-5448-48c3-8ee5-a2da73195857)
![alteractal24](https://github.com/user-attachments/assets/f980d75d-ef7c-48ee-b9ca-3d6d9d2e6dbb)
Unknown exact date, somewhere between 1.5.0 and 1.8.0. Shows the old location of IB gy flag.

![ss12](https://github.com/user-attachments/assets/37fbe414-5166-4c1d-a876-b04d74c539e0)
I found this old screenshot, from blizz pre-release AV. Seems to display Stronghoof more near mid, it's possible he used to be there when graveyard was closer to mid.


---
Missing NPCs that do not exist in any sniff, not even korraks revenge.

South bunker with a bunch of NPCs:
![image](https://github.com/user-attachments/assets/73c7d58c-1422-4034-b283-ea8ec5b1a7c6)

https://youtu.be/nUyqdbm7zho?si=vmYPy1MjoIRlcWSS&t=105
Druid pulling Dardosh, whith two additional melee guards around flag. Closed sides, so at least patch 1.8+.
![image](https://github.com/user-attachments/assets/eaa4e5ff-58ae-4745-bffe-48251b94d597)
Image from the above video.

Dun Baldar North bunker with melee guards inside:
![image](https://github.com/user-attachments/assets/ba98a63f-143c-40b6-bcf8-483602b06254)
Unknown video, but was from after patch 1.8.0.

West Frostwolf Tower with melee guards inside:
![image](https://github.com/user-attachments/assets/80bdf917-0227-416f-8c75-f71a9816cf21)
Screenshot from this video:
https://youtu.be/P6644EITQwg?si=r5--lW8g3WN7fFYN&t=43

![WoWScrnShot_103005_122036](https://github.com/user-attachments/assets/558a7db5-b82e-424b-b8ac-77756fde1990)
East Frostwolf Tower with melee guards showing. Dated October 30 2005.

![WowScreenshot8](https://github.com/user-attachments/assets/1e11f889-e99f-41a5-b99e-0b2b5902083a)
East and West Frostwolf Tower with melee guards. Unknown date.

NPCs around TP:
![image](https://github.com/user-attachments/assets/4bd41669-fc46-43ba-bf25-d3b3711f10e9)
Dated 15 June 2005.
![image](https://github.com/user-attachments/assets/04ecd3bc-4284-45fb-bb19-09a0fe6751cb)
Unknown date.
![image](https://github.com/user-attachments/assets/66dc14b6-a33b-4e1e-9223-6647d7db4c82)
Pre-release patch 1.5.0.
![0722222](https://github.com/user-attachments/assets/7f16af4e-d814-46fc-ad1c-715476199ccc)
Pre-release patch 1.5.0. Does match from later release vanilla: https://youtu.be/uYYYbZ51O8A?si=T0YPDJoT8BOD5XOt&t=670
![1266](https://github.com/user-attachments/assets/a13f1c07-6803-467e-b128-156f9f254075)
Blizz image pre-release patch 1.5.0. 

![6](https://github.com/user-attachments/assets/5e052b18-01cd-48b6-b96d-05b3ef88fe87)
Unknown horde tower pre-release 1.5.0.

NPCs infront of SH bunker:
![image](https://github.com/user-attachments/assets/3529f6c9-15ba-4f24-b125-90613de6b4f8)
Unknown video and unknown date. I only have the image.
I do however have the spawns from SoM sniffs, so we all good here.

![ss18](https://github.com/user-attachments/assets/994e5bd3-3f2a-4000-a03b-f540eaa98788)
East Tower with melee guards, image from blizz pre-release.

Horde entering SH bunker on a fresh new AV, to be meet by tons of melee NPC guards:
https://youtu.be/uYYYbZ51O8A?si=ktseTo4jnDmLmu31&t=1494

---
Weird flag location of Tower Point.
![213 - vtap4](https://github.com/user-attachments/assets/6b14c59d-8c20-45f2-811e-03ebfe59fe1d)
Unknown date, but according to patch notes 1.10.0: "Tower Banners in Alterac Valley can no longer be used through walls.".
So perhaps some flags used to be very close to walls before.


![03040404](https://github.com/user-attachments/assets/b20c63c1-bf0d-4a2f-ac9e-f42b780b20a2)
Ice Blood Tower with flag closer to wall. Pre-release patch 1.5.0.

West Frostwolf Tower with flag closer to the wall:
![WoWScrnShot_071305_071717](https://github.com/user-attachments/assets/e66a58bf-3893-46d1-b98a-044c7eeae1eb)
![WoWScrnShot_071305_072514](https://github.com/user-attachments/assets/266f3fc9-6e51-4452-b343-165cbcf40af2)
Dated July 13 2005.

I have the spawns of the old tower flags from wowemu, seems accurate.
However it turns out bunker flags used to be closer to walls too:
![mpc-hc64_ZjsdV2zrum](https://github.com/user-attachments/assets/1ee8132f-97fe-46f7-b125-8e972814137d)
From this video https://streamable.com/0akj7c at 5:05.
Video dated 2005 08 14

But it seems bunker flags were changed sometime early as well, below is a from a dated 2005 08 11 video that still uses the old graveyards but has Icewing bunker with new flag position:
![mpc-hc64_4rX7LMO8YU](https://github.com/user-attachments/assets/59723641-3878-48e3-ba1f-dd2ba2824980)
![mpc-hc64_ILIsETFxkS](https://github.com/user-attachments/assets/4036cf3b-9bb7-464c-95c0-c6b045e8670b)


---
![0444444](https://github.com/user-attachments/assets/3e8900cd-aad8-42e5-bb53-bec4628de7ca)
Old Horde Spirit Guide location in cave. Pre patch 1.9.3, as the patch notes says "The default graveyards in Alterac Valley for each side were moved further up the tunnels away from the battlegroud[sic] exit. This should prevent accidentally leaving the battleground."

---
![wow_screen_20060117_204306_466_00061](https://github.com/user-attachments/assets/ebad490e-1e78-4da1-b4de-3d80a59c12a4)
Image showing Snowfall Graveyard between 1.8 and 1.10. Dated 15 Jan 2006 Looks like blizz just moved the pack that stands near bridge in Korrange revenge, to further back when they remade snowfall graveyard in patch 1.8.

---
Korrak pre 1.8 is currently not in the correct location.
Pre 1.8 it appears he roams mid.

![WoWScrnShot_062105_165015](https://github.com/user-attachments/assets/08539627-dd69-43a9-9e5a-b6f24c590291)
Image from 21st of june 2005 (patch 1.5.1).

![aerial_h](https://github.com/user-attachments/assets/496b07cd-1fd5-4aab-a9cb-07e00522042e)
Image from blizz, probably from pre-release patch 1.5.0

https://www.youtube.com/watch?v=-7RBF8cYZRs
Video from 27 06 2005.

![alteractal29](https://github.com/user-attachments/assets/85ca01a3-5220-4bee-bcfd-71b5817e4ece)
![alteractal28](https://github.com/user-attachments/assets/a794710f-82a1-4133-98fd-491c0245d2ba)
Unknown date, but shows him in combat, very close to the location of the previous screenshots.

![brave_vV6asBbMep](https://github.com/user-attachments/assets/f26f0d2a-3005-41ca-b4ee-b6e911e67199)
Korrak out of combat, from this video: https://archive.org/details/ctz666_-_Backstab_5_Rogue_2005-06-20_4.35

---
Herbs in Alterac Valley:
![image](https://github.com/user-attachments/assets/14b9c195-b542-426d-bec4-f58d9b177451)
Seen in this video https://streamable.com/0akj7c at 0:20.
Appears to be a Golden Sansam.

![brave_5cXFpBWfcc](https://github.com/user-attachments/assets/8c507389-ef5e-434e-88f3-768fde94e66e)
Golden Sansam in Field of Strife, from this video:
https://archive.org/details/ctz666_-_Backstab_5_Rogue_2005-06-20_4.35

---
Rich Thorium Veins:
![brave_aykYstC8mM](https://github.com/user-attachments/assets/a15f93f8-e106-442b-a10e-4832707adbdc)
![brave_kzTroCIW8Y](https://github.com/user-attachments/assets/9b542b98-2611-40fc-a61f-312afbbabc3d)
From this video: https://archive.org/details/ctz666_-_Backstab_5_Rogue_2005-06-20_4.35


---
Fix tower/bunker flag spawns, currently they all share either the horde or alliance spawn, however it should have separate spawns:

![WoW_lGbH38iJGy](https://github.com/user-attachments/assets/d95c44b4-16b0-463a-b05f-1852577fbc68)
![WoW_Jvb7BF1m5Y](https://github.com/user-attachments/assets/e642b604-4c01-4212-8a11-a52e185f18eb)
![WoW_0f027TGUXw](https://github.com/user-attachments/assets/ccd8c99c-bed3-4bcc-a7f3-9a4f2d0ca05b)
![WoW_7RQhug8hsZ](https://github.com/user-attachments/assets/aad4a388-dbdb-4617-87c4-119317d1dcbb)

---
Several harpie spawns missing here, that exist in sniff:
![WoW_p2HQb3jxHk](https://github.com/user-attachments/assets/4affba9d-af76-4322-96f3-697b94b5baef)

--- 
Missing NPCs in Alliance Lumber Yard:
![brave_p8BWlE2oM9](https://github.com/user-attachments/assets/727ef895-c00e-49da-a90e-ef6e0b844fe2)
![brave_4T21wzvUhf](https://github.com/user-attachments/assets/2e50fa1e-d589-4ec0-925a-bd746a73acea)
![brave_KnqxNAhqqO](https://github.com/user-attachments/assets/a2ef1ac8-f5eb-4af7-ad97-687927d46455)
Seen in this video: https://archive.org/details/ctz666_-_Backstab_5_Rogue_2005-06-20_4.35

---
Old Alliance cave Spirit guide location:
![brave_dnyEKUc9fT](https://github.com/user-attachments/assets/66303dc5-43bf-4c89-8242-ee73e1f50c40)
From this video: https://archive.org/details/ctz666_-_Backstab_5_Rogue_2005-06-20_4.35

Daribon committed:
Add missing AV objects from sniff.

- Added various missing objects from sniff, that should exist in all versions, but was missing in Korrak's revenge sniffs.
- Deleted duplicate chairs, only chairs found in sniff are left.
- Deleted custom cauldron, replaced with campfire found in sniff.
- Removed custom campfire in front of Drek fortress.
- Removed custom entry Gnomish Toolbox in gameobject_template, was incorrectly used for Steamsaw loot. However a sniffed spawn for Steamsaw already exists.

moyashiparty commented:
> Weird object's all around Drek'thar and Vanndar in sniffs:

Dunno if it helps but I have an old screenshot that shows the documents around Vann, I believe around Patch 1.8.3 or 1.8.4. No idea if they spawned on death or were always present though:
![WoWScrnShot_120505_030022](https://github.com/user-attachments/assets/02b99a53-9ef8-4c4b-97ea-33239835a3bc)

Daribon commented:
> [moyashiparty](/moyashiparty)

I've realized that they spawn exactly whenever Drek'thar or Vanndar dies.
See the following video: https://www.youtube.com/live/9lBW-Ab1CMs?si=KpLYlc-1GufYXbVR&t=3214

Looks like left over objects before Alterac Valley was released, when it was still a zone and it was possible to do stuff after killing Drek or Vanndar.

Daribon committed:
Delete duplicate NPCs.

Removed duplicate rams and some spawns that shouldn't exist.

Daribon committed:
Update some NPCs to use co2

Daribon committed:
Add missing co2 NPC spawns.

Added one missing ram co2 spawn and three missing frostwolf co2 spawns. Updated the frostwolf at fw gy to use co2.

Daribon committed:
Fix query

Daribon committed:
Corrected bunker/tower banners

Daribon committed:
Correct bunker/tower flags

Daribon committed:
Correct GY flags for 1.12.1

From classic and SoD sniffs.

Daribon committed:
Old sniffed pre 1.8 GY spawns.

Sniffed spawns from original WoWemu DB.

Daribon commented:
Realized I missed SF GY, gonna do that later for 1.12.1 and pre 1.8. I have sniffed spawns for both.

Daribon committed:
Sniffed old herb spawns pre 1.8

Sniffed herb spawns from original WoWEmu DB.
Needs pooling.

Daribon committed:
Add missing building fires.

Daribon committed:
Pooling for mining nodes, credits to NickTyrer.

Daribon commented:
Added the fires that should spawn after Vanndar dies.
Video showing how it spawns upon death:
https://youtu.be/D2yDzcer7Ig?si=K4ld0-vpbBAxOELp

Needs script. Oddly enough, no such "death fires" exist on horde side.

Daribon commented:
I'm back working on this.

Managed to find some more images:
Melee guards in west fw tower, in the distance while tower burning:
<img width="1024" height="768" alt="030 - 2kBvr" src="https://github.com/user-attachments/assets/40051785-0737-47d2-a41f-940fd1ceaa92" />


Deathstalker Agent NPC corpse in Tower Point??
As far as I know this NPC only existed on the PTR, but below image is not on PTR. Also do note, old flag location in TP tower: ![213 - vtap4](https://github.com/user-attachments/assets/6b14c59d-8c20-45f2-811e-03ebfe59fe1d)

https://www.wowhead.com/classic/npc=13155/deathstalker-agent

In the original wowemu DB (which was mainly based on sniff) there is a Deathstalker Agent spawned right next to Commander Louis Phillips. Which makes above even weirder, since as far as I know, wowemu didn't sniff on PTR (but could have been possibly been spawned by hand, which is weird too since I have not yet seen any hand spawning in AV in that DB):
<img width="1198" height="711" alt="Screenshot_20260114_234734" src="https://github.com/user-attachments/assets/e912d9cf-34f9-46c4-ad0d-b6204dc535d9" />
Image is a mix of wowemu and vmangos DB.

Above image does show something weird that I've noticed in towers in wowemu as well, it shows Wing Commander Slidore ontop of the tower. Now that could have been someone just pulling it up to the tower and it getting caught in sniff.

But! When I check wowemu for West Frostwolf Tower, Wing Commander Ichman is also spawned ontop of tower. In this case even a trigger is located there. An old thottbot comment says "Ichman is actually standing in the bottom of the tower, but you have to go to the top of the tower to find him." Which indicates that possibly they moved him in some patch to bottom, but forgot to move his trigger for the quest, only to later move the trigger to the bottom as well.
<img width="1198" height="711" alt="Screenshot_20260114_234947" src="https://github.com/user-attachments/assets/423a9567-c115-435f-b762-d58bddbf6456" />

Frostwolf Herald in wrong location for old patch:
<img width="1280" height="1024" alt="280 - jZemy" src="https://github.com/user-attachments/assets/9340da76-d2cd-4767-99d6-1b4af528f1e7" />
Wowemu has this sniffed, see target:
<img width="1198" height="711" alt="Screenshot_20260114_235733" src="https://github.com/user-attachments/assets/9b63c32e-4592-42b3-8a07-a5c57d845842" />
(Left is vmangos, right is wowemu)

Seasoned Guardsman standing outside Icewing Bunker, dated February 12 2006:
![WoWScrnShot_021206_000942](https://github.com/user-attachments/assets/41fa46d6-38b0-4140-9ee7-bcffa8435542)

Something is off with the old spawning of Frostwolf Keep, images from unknown date:
![3777](https://github.com/user-attachments/assets/28abf9d2-4524-4396-95aa-f12affe25871)
This matches up with wowemu spawning, for example in below image the wolves in the front are from wowemu but the ones in the back are vmangos:
<img width="1858" height="977" alt="Screenshot_20260115_002313" src="https://github.com/user-attachments/assets/f63fc4cd-83d7-45ae-a839-3ad17b30e505" />

Dreamfoil in Alterac Valley, screenshots from this video: https://youtu.be/CJ1-LeqJXFc?si=XCAlApLprP_BA_AS&t=158
<img width="1924" height="1344" alt="Screenshot_20260116_235526" src="https://github.com/user-attachments/assets/a297d192-5ada-4833-a287-27807f4bbcbd" />
<img width="1924" height="1344" alt="Screenshot_20260116_235350" src="https://github.com/user-attachments/assets/2d94870c-085a-454e-83c4-b3a24d83ad95" />

Video showing the amount of of melee guards that used to exist in bunker:
https://youtu.be/PwupuiCBRhs?si=O-7q6Sizp4kuinq9&t=419
<img width="1036" height="533" alt="Screenshot_20260115_223855" src="https://github.com/user-attachments/assets/9a5219a2-7974-4e52-8853-cd5cfeaabbc4" />

Video showing some of the guards at lumber camp:
https://youtu.be/sKiumzczimI?si=UDcrV-kQ3NeJkErw&t=765
Another video showing almost all guards at lumber camp:
https://youtu.be/AHv1KJlapKU?si=Fa4Zs2aowE39hyL5&t=207

<img width="1303" height="829" alt="Screenshot_20260115_181748" src="https://github.com/user-attachments/assets/1ebdfe29-214d-46ad-9245-93338ab2292d" />
<img width="1303" height="829" alt="Screenshot_20260115_181738" src="https://github.com/user-attachments/assets/893fd607-8ad7-498c-bfb7-7417cdf1182e" />
<img width="1303" height="829" alt="Screenshot_20260115_181713" src="https://github.com/user-attachments/assets/6e430e76-76fc-4dbd-ad1a-b46ac970e2b2" />

Video showing two melee guards inside east fw tower:
https://youtu.be/Ds3g7a-7q8g?si=yuWboMglft8A_r-L&t=448
<img width="1434" height="1065" alt="Screenshot_20260115_225642" src="https://github.com/user-attachments/assets/db17b584-0356-463b-9d8c-95784a3899ef" />
<img width="1434" height="1065" alt="Screenshot_20260115_225659" src="https://github.com/user-attachments/assets/9c3cda69-17d5-4d1c-b3e8-05d4a02c4127" />

Can't remember what video this screenshot is from, but the only thing interesting in the video was the two guard spawns shown below:
<img width="1303" height="829" alt="Screenshot_20260115_191310" src="https://github.com/user-attachments/assets/7dbb9a56-c8cb-4083-9dd7-c94abb4b45af" />

moyashiparty commented:
Not sure if it helps with anything cause it doesn't show a whole lot, but I found this video on an old hard drive which shows a bit of AV:
https://www.mediafire.com/file/uqnndjh9uov8uoe/Goukipvp4test.wmv/file
```
Goukipvp4test.wmv
Modify: 2005-07-12 19:27:16.000000000 +1000
```
Which puts it around 1.5.0-1.5.1.

ratkosrb commented:
streamable reupload https://streamable.com/v17g19

Daribon commented:
> Not sure if it helps with anything cause it doesn't show a whole lot, but I found this video on an old hard drive which shows a bit of AV: https://www.mediafire.com/file/uqnndjh9uov8uoe/Goukipvp4test.wmv/file
> 
> ```
> Goukipvp4test.wmv
> Modify: 2005-07-12 19:27:16.000000000 +1000
> ```
> 
> Which puts it around 1.5.0-1.5.1.

Sadly it doesn't contain footage of the areas I need, that particular area I got everything I need from either Season of Mastery, Korraks Revenge or wowemu.

I need videos (or images) showing IB GY, TP and FW gy (pre capture so with both commanders and LTs up)
I do have the spawns (that are sniffed) from wowemu, which do look very accurate but I just need to confirm that it wasn't captured during combat movement.

I also need TP on patch 1.5 to confirm if Deathstalker Agent actually did make it out of beta (since for whatever reason they do appear on wowemu, which is almost completely unfiltered parsed sniff, it is however a slight possibility that wowemu team started sniffing on av PTR)

Anything pre 1.9 is also acceptable for other AV locations, for example I need more images or videos showing the lumber mill locations in AV.

With or without additional evidence, I’m going to wrap this up soon. Getting this finished matters more, since it’s the last obstacle to somewhat blizzlike gameplay (all other content such as raids and dungeons are somewhat blizzlike). In the case more proof does show up afterwards, new PR can be created.

moyashiparty commented:
I found some more screenshots of around 1.8.3-1.8.5, again not sure it will help but you never know:

October 2005
<img width="800" height="600" alt="WoWScrnShot_102005_191650" src="https://github.com/user-attachments/assets/7f34afe1-e41e-43d4-8fc7-63c990b0f142" />

November 2005
<img width="1280" height="960" alt="WoWScrnShot_112905_012006" src="https://github.com/user-attachments/assets/93216f1d-0b2e-43f8-8d7e-b3c02f4abc33" />

<img width="1280" height="960" alt="WoWScrnShot_112905_014426" src="https://github.com/user-attachments/assets/64edb5f3-252b-487d-9365-ac437b4428a2" />

December 2005

<img width="1280" height="960" alt="WoWScrnShot_120205_201246" src="https://github.com/user-attachments/assets/e0417b3b-9728-410c-834b-9a4c230dc2a1" />
